activation/poet.go

```diff
@@ -63,6 +63,9 @@ func withCustomHttpClient(client *http.Client) PoetClientOpts {
 
 // NewHTTPPoetClient returns new instance of HTTPPoetClient connecting to the specified url.
 func NewHTTPPoetClient(baseUrl string, cfg PoetConfig, opts ...PoetClientOpts) (*HTTPPoetClient, error) {
+	// TODO(brozansk): Take a logger and use it instead of global logger
```

Comment:
The `retryablehttp.Client` also takes a `Logger` field. Maybe we can also init that with the logger passed to `NewHTTPPoetClient`?